Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -680,66 +679,6 @@ protected void StartInputTimer(bool isInput = false)
 
         #region MixedRealityInputHandlers
 
-        /// <summary>
-        /// Used for click events for actions not processed by pointer events
-        /// </summary>
-        /// <param name="eventData"></param>
-        public void OnInputUp(InputEventData eventData)
```

Comment:
I removed OnInputUp/OnInputDown from interactable because I don't see a case where OnPointerDown / OnPointerUp does not handle this, and because I couldn't think of a way to not respond to OnInputUp from PokePointer without specifically ignoring that type

Reply:
Agreed. If it breaks something at least we will understand the use case that is meant to cover.

Reply:
Okay, I think I figured out why @killerantz  is listening to OnInputUp. Based on the comments it seems like previously the only way to globally listen for inputs was to listen for OnPointerUp events, and then fire OnClicked. Now, Pointer events are also dispatched globally. Since Interactable is registered as a global listener, it should also be able to receive all global pointer events.